dotNETStandard/HealthVaultProviderManagementPortal/HealthVaultProviderManagementPortal/Views/Patient/Index.cshtml

```diff
@@ -67,3 +66,18 @@
     }
 }
 
+@helper TaskTime(TimelineEntryViewModel task)
```

Comment:
Is this helper likely to be used anywhere else? Perhaps a different progress or summary view in the future? If so, maybe this helper be moved into the `App_Code` folder?